src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/model/caserecord/ExceptionRecord.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.model.caserecord;
+
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.model.CaseRecord;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.model.Envelope;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDataContent;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.ccd.client.model.Classification;
+
+public class ExceptionRecord implements CaseRecord {
+
+    private final Envelope envelope;
+    private final CaseDetails caseDetails;
+
+    public ExceptionRecord(Envelope envelope, CaseDetails caseDetails) {
+        this.envelope = envelope;
```

Comment:
Null checks ?

Reply:
Envelope already has been accessed multiple times before reaching this point. Should I just add null check for it at the beginning anyway?